README.md

```diff
@@ -140,6 +141,30 @@ client = Client.new(url, schema_path: 'config/your_graphql_schema.json')
 client.schema.dump! # you only need to call this when graphql schema changes
 ```
 
+### Preloading schema once
+
+Even if caching the schema on disk, instantiating `Graphlient::Client` often can be both time and memory intensive due to reading in the schema for each instance. This is especially true if the schema is a large file. To get around these performance issues, instantiate your schema once and pass it in as a configuration option.
+
+One time in an initializer
+
+```ruby
+$schema = Graphlient::Schema.new(
```

Comment:
Nit: I would get rid of `$`, that implies some kind of global that's rarely used in Ruby.